README.md

```diff
@@ -63,16 +63,15 @@ npm install @google-cloud/functions
 // eslint-disable-next-line node/no-missing-require
 const {CloudFunctionsServiceClient} = require('@google-cloud/functions');
 
-// TODO(developer): replace with your prefered project ID.
-// const projectId = 'my-project'
-
```

Comment:
Thanks for submitting this review! Unfortunately, this method of authentication is just the way we do it across all of our client libraries - we do want to encourage users to authenticate beforehand.

Reply:
I'm not sure I understand this comment. This is using the default application credentials (which requires users to authenticate beforehand).

Reply:
Right. However, we leave this section commented out so users can input their project ID if they want to.